HW0/P5/P5.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+import matplotlib.pyplot as plt
+# Get lots of values for N
+N = np.array(range(2, 1000, 10))
+# Calculate the logs
+logs = np.ceil(np.log2(N))
+# Plot the figure
+plt.figure(1)
+plt.scatter(N, logs, c='red', label='Infinite Employees')
+plt.scatter(N, N-1, c='blue', label='One Employee')
+plt.xlabel("Number of bags")
+plt.ylabel("Time Needed (sec)")
+plt.yscale('log')
```

Comment:
Nice choice using the log scale!  You can see the difference in shape between the two curves while still looking at a lot of data points.  (I just did it on a linear scale and had to zoom in to see the shape of the infinite employee curve.)